src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -688,6 +721,95 @@ private static string GetSearchText(string parameterString)
             return parameterString.Substring(0, lastIndexOfSearchInSwitch);
         }
 
+        /// <summary>
+        /// Gets the names of the package or packages that are selected in the VsHierarchy.
+        /// </summary>
+        /// <returns>
+        /// A string array containing the list of selected packages in the same hierarchy.
+        /// If only one package is selected, the array will contain one package name, otherwise, it will be a list of package names.
+        /// If the selected packages are across multiple hierarchies or if no packages are selected, an empty array will be returned.
+        /// </returns>
```

Comment:
Should warn that `null` may be returned, and why.

Is NuGet using nullable reference types anywhere?

Reply:
Thanks! I added a comment. The VS integration part of NuGet is not using nullable reference types.